Examples/Tests/radiation_reaction/test_const_B_analytical/inputs

```diff
@@ -0,0 +1,66 @@
+#Automatically generated inputfile
+#Run check.py without arguments to regenerate
+#
+
+max_step = 80
+amr.n_cell = 80 80 80
+amr.max_level = 0
+amr.blocking_factor = 8
+amr.max_grid_size = 8
```

Comment:
That's why the test is so slow :). You can read this page https://warpx.readthedocs.io/en/latest/running_cpp/parallelization.html for information on these parameters. I'd suggest using the following parameters:
```
amr.n_cell = 64 64 64
 amr.blocking_factor = 32
 amr.max_grid_size = 64```
and recompute the `max_step` accordingly (probably `max_step = 64`).

Reply:
You are right. I've just changed the parameters.